src/integrations/buttons.js

```diff
@@ -14,7 +14,7 @@ function isVmUp (vm) {
 }
 
 function addVmManageGpuButton () {
-  getPluginApi().addDetailPlaceActionButton(entityTypes.vm, entityTypes.hostDevices, msg.vmManageGpuButton(), {
+  getPluginApi().addDetailPlaceActionButton(entityTypes.vm, entityTypes.vmDevices, msg.vmManageGpuButton(), {
```

Comment:
I cannot see the button there (nor in the host devices tab).  Perhaps a problem on my side?

Reply:
@ljelinkova  would this be solved by https://github.com/oVirt/ovirt-engine/pull/173?

Reply:
Yes